src/common/admin_socket.cc

```diff
@@ -443,7 +443,7 @@ int AdminSocket::register_command(std::string command, std::string cmddesc, Admi
 {
   int ret;
   m_lock.Lock();
-  if (m_hooks.count(command)) {
+  if (m_hooks.find(command) != m_hooks.end()) {
```

Comment:
*  This function only makes sense for multimaps; for map the result will
       *  either be 0 (not present) or 1 (present).
       */